code/lib/cli/src/automigrate/fixes/vta.ts

```diff
@@ -0,0 +1,60 @@
+import { dedent } from 'ts-dedent';
+import { updateMainConfig } from '../helpers/mainConfigFile';
+import type { Fix } from '../types';
+import { getStorybookVersionSpecifier } from '../../helpers';
+
+const logger = console;
+
+interface Options {}
+
+/**
+ */
+export const vta: Fix<Options> = {
+  id: 'visual-testing-addon',
+
+  versionRange: ['<8.0.7', '>=8.0.7'],
+
+  async check({ mainConfig }) {
+    const hadAddonInstalled = !!mainConfig?.addons?.find((addon) =>
```

Comment:
Use the helper function `getAddonNames` from '../helpers/mainConfigFile' to extract the addon names